### PR TITLE
Redesign send tokens

### PR DESCRIPTION
**Pull request by nicob21**
Related to: https://app.asana.com/0/1165477653959782/1200431032618769/f

Forward a callback function to the API to notify front-end when the user confirm the transfer on its Ledger

### CODE REVIEW

File: src/api/ledger/transferUsingVaultIdAndStarkKey.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
   const url = dvf.config.api + '/v1/trading/w/transfer'
   const createSignedTransferTx = makeCreateSignedTransferTxLedger(dvf)(path)
-  const json = await dvf.createTransferPayload(transferData, feeRecipient, createSignedTransferTx)
-  return post(url, { json })
+  const json = await dvf.createTransferPayload(
+    transferData,
+    feeRecipient,
+    createSignedTransferTx
+  )
+  return post(url, { json }, walletSignCb)
```
Comment (Neozaru) [inline src/api/ledger/transferUsingVaultIdAndStarkKey.js:18]: I think technically you can say that the user has signed after `dvf.createTransferPayload` is successful (maybe you want to call you callback at this post ?)
The `post` is just the HTTP request, which as far as I understand will resolve by itself when successful (not sure how `walletSignCb` plays out passed like this)

Comment (nicob21) [reply src/api/ledger/transferUsingVaultIdAndStarkKey.js:18]: Yes you are completely right, I'll make the change, thanks

**Review by Neozaru — APPROVED** [review]

**Comment by Neozaru** [comment]
@nicob21 I can see that tests passed on FE for this task already. Does that mean this PR is not needed anymore or should it be merged ?

**Comment by nicob21** [comment]
Yes correct, this PR is not required anymore. We can close.

**Pull request closed without merging**